pkg/olm/packagemanifestlist.go

```diff
@@ -36,15 +39,28 @@ func ListPackageManifest(
 
 	glog.V(100).Infof(logMessage)
 
-	pkgManifestList, err := apiClient.PackageManifestInterface.PackageManifests(nsname).List(context.TODO(),
-		passedOptions)
-
+	var pkgManifestList *v1.PackageManifestList
+	err := wait.PollUntilContextTimeout(
+		context.TODO(), 2*time.Second, time.Minute, true, func(ctx context.Context) (bool, error) {
+			var err error
```

Comment:
@empovit so in that case, why to use 2 functions?
Becuase of this bug, wouldn't it make more sense to use only ListPackageManifestWithTimeout()?

Reply:
`packagemanifestlist.go` seems like an "infrastructure" file, it may be used by multiple test. Also, changing the "contract" of a function isn't a good practice. In this case you're silently adding polling, with hard-coded times, and without communicating the changed behavior in the function's name. If you think we don't need two versions then at least rename it.

Reply:
@empovit done